mollie.php

```diff
@@ -1203,6 +1220,128 @@ public function getTabs()
         ];
     }
 
+    public function hookActionAdminOrdersListingFieldsModifier($params)
+    {
+        if (isset($params['select'])) {
+            $params['select'].= ' ,mol.`transaction_id`';
+        }
+        if (isset($params['join'])) {
+            $params['join'].= ' LEFT JOIN `'._DB_PREFIX_.'mollie_payments` mol ON mol.`order_id` = a.`id_order`';
+        }
+        $params['fields']['order_id'] =  [
+            'title' => $this->l('Resend payment link'),
+            'align' => 'text-center',
+            'class' => 'fixed-width-xs',
+            'orderby' => false,
+            'search' => false,
+            'remove_onclick' => true,
+            'callback_object' => 'mollie',
+            'callback' => 'resendOrderPaymentLink'
+        ];
```

Comment:
I dont like seing plain sql in controllers, hooks - atleast create a service for this functionality you are working here.